blynk_ncp/src/zephyr_lib.c

```diff
@@ -21,14 +21,24 @@ LOG_MODULE_REGISTER(blynk_ncp, CONFIG_BLYNK_LOG_LVL);
 #include <blynk_ncp/blynk_ncp.h>
 #include <BlynkRpcClient.h>
 
-#ifndef BLYNK_THREAD_PRIO
-#warning BLYNK_THREAD_PRIO not configured
-#define BLYNK_THREAD_PRIO     (CONFIG_NUM_PREEMPT_PRIORITIES - 1)
+#ifndef CONFIG_BLYNK_THREAD_PRIO
+#warning CONFIG_BLYNK_THREAD_PRIO not configured
+#define CONFIG_BLYNK_THREAD_PRIO     (CONFIG_NUM_PREEMPT_PRIORITIES - 1)
 #endif
 
-#ifndef BLYNK_THERAD_STACK_SIZE
-#warning BLYNK_THERAD_STACK_SIZE not configured
-#define BLYNK_THERAD_STACK_SIZE          (512)
+#ifndef CONFIG_BLYNK_THERAD_STACK_SIZE
```

Comment:
These configs are already set to default using KConfig, we don't need to duplicate them here.